http-utils/src/main/java/com/facebook/airlift/http/utils/jetty/ConcurrentScheduler.java

```diff
@@ -0,0 +1,68 @@
+package com.facebook.airlift.http.utils.jetty;
```

Comment:
This should be done using git-mv not regular mv

Reply:
Thanks, I will keep this in mind. It seems like ``git show`` identified the move operation and correctly displays the changes. Is there anything missing and should I redo?

Reply:
Github can be getting confused sometimes

Reply:
Ah, my bad.